apps/console.py

```diff
@@ -660,6 +662,13 @@ async def do_advertise(self, params):
         else:
             self.show_error('unsupported arguments for advertise command')
 
+    async def update_timer(self):
+        if self.top_tab == 'local-services':
+            self.update_local_services(self.device.gatt_server.attributes)
+            self.ui.invalidate()
+            await asyncio.sleep(1)
+            await self.update_timer()
```

Comment:
I suggest using a loop instead, only for readability purpose.
Is tail call elimination supported for asynchronous function ?